rackcity/views/site_views.py

```diff
@@ -51,175 +64,200 @@ def datacenter_create(request):
             status=HTTPStatus.BAD_REQUEST,
         )
     try:
-        new_datacenter = serializer.save()
+        new_site = serializer.save()
     except Exception as error:
         return JsonResponse(
             {
                 "failure_message": Status.CREATE_ERROR.value
-                + "Datacenter"
+                + "Site"
                 + GenericFailure.ON_SAVE.value,
                 "errors": str(error),
             },
             status=HTTPStatus.BAD_REQUEST,
         )
-    log_action(request.user, new_datacenter, Action.CREATE)
+    log_action(request.user, new_site, Action.CREATE)
     return JsonResponse(
-        {
-            "success_message": Status.SUCCESS.value
-            + new_datacenter.abbreviation
-            + " created"
-        },
+        {"success_message": Status.SUCCESS.value + new_site.abbreviation + " created"},
         status=HTTPStatus.CREATED,
     )
 
 
 @api_view(["POST"])
 @permission_required(PermissionPath.ASSET_WRITE.value, raise_exception=True)
-def datacenter_delete(request):
+def site_modify(request):
```

Comment:
modify endpoint was moved from bottom of file to here so it looks like it replaced the delete endpoint but that's a LIE - might be easier to look at changes in PyCharm

Reply:
Github is brazy